ledger/puzzle/epoch/src/synthesis/helpers/mod.rs

```diff
@@ -345,6 +345,23 @@ pub(crate) mod tests {
 
     const ITERATIONS: u64 = 25;
 
+    #[test]
+    fn test_longest_sequence() {
+        let instruction_set_weights = instruction_set::<CurrentNetwork>();
+        let mut longest_sequence: usize = 0;
+        for isw in instruction_set_weights {
```

Comment:
Could do this via an iterator for more clarity.
Something like: `instruction_set_weights.iter().map(|(_, sequence) sequence.len()).max()`